### PR TITLE
lab03 & add gradle

### PR DESCRIPTION
**Pull request by derenlei**
Complete the F17_lab03.md and add the gradle which connects to the build.xml (issue #23 ). However the gradle might need to be rewritten to support future code adding.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1 +1,34 @@
-ant.importBuild 'build.xml'
\ No newline at end of file
+apply plugin: 'java'
+apply plugin:'application'
+mainClassName = "edu.ucsb.cs56.projects.utilities.credit_card_validator"
```
Comment (jaeaster) [inline build.gradle]: This line should be:
`mainClassName = "edu.ucsb.cs56.projects.utilities.credit_card_validator.Menu"`

File: build.gradle [hunk 2]
```diff
@@ -1 +1,34 @@
...
+
```
Comment (jaeaster) [inline build.gradle]: On the following line add this:

`sourceSets.main.java.srcDirs` = ['src']

This says "Hey gradle, go look for my main class specified above in the `src` directory"

By default, gradle looks for a directory named `src/main/java/` for the start of your class path.

**Review by jaeaster — CHANGES_REQUESTED** [review]

**Comment by jaeaster** [comment]
Can you make a separate pull request for the gradle changes? This PR should only include the commit for the lab 03 assessment.

You can do this like so:
```
git checkout -b addGradle
git push origin addGradle
git checkout master
git reset --hard HEAD~4
git push -f origin master
```

**Comment by derenlei** [comment]
I used the code above and this pull request is automatically closed. How can I do separate pull requests?

**Comment by jaeaster** [comment]
@derenlei Don't worry about the pull requests for now we'll figure it out next week

**Pull request closed without merging**